src/js/game_logic.js

```diff
@@ -32,8 +32,8 @@ const guessElement = document.getElementById('rhyme_guess')
 const wrongElement = document.getElementById('incorrect')
 const hint = document.getElementById('hint')
 const timerDisplay = document.getElementById('timer')
-const startButton = document.getElementById('start')
-const restartButton = document.getElementById('restart')
+const startButton = document.querySelector('.header_button_start')
+const restartButton = document.querySelector('.header_button_restart')
 
```

Comment:
For anyone curious about `getElementById` vs `querySelector`, check out [this post on StackOverflow](https://stackoverflow.com/questions/26848289/javascript-queryselector-vs-getelementbyid)!

Reply:
I choose to go with `querySelector` due to the fact that performance isn't an issue for something like this. Now if it gets monolithic there can potentially be an issue but I have worked in bigger scaled applications and that doesn't seem to cause bottlenecks if imposed correctly but its all personal preference. Also with `querySelector` or `querySelectorAll`, you can target things like you would in CSS so for example `document.querySelector(".button[disabled=true]")`. 

__*tldr:*__ In the end its personal preference depending on how cognitive you are on performance and making sure it functions across browser by crosschecking in [caniuse](https://caniuse.com/#feat=queryselector).